src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp

```diff
@@ -485,7 +485,7 @@ namespace OpenMS
       category_set << String(it->second.category).toQString();
     }
     QStringList category_list = category_set.toList();
-    qSort(category_list);
+    std::sort(category_list.begin(), category_list.end(), std::less<QString>());
```

Comment:
Hi, would
```suggestion
    std::sort(category_list.begin(), category_list.end());
```
also work? I think QString defines an operator<